queries.sql

```diff
@@ -31,3 +31,56 @@ WHERE neutered = true;
 SELECT *
 FROM animals
 WHERE  weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Transactions and modifications
+
+BEGIN;
+UPDATE animals SET species = 'undefined';
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE animal_name LIKE '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE animal_name IN ('Charmander', 'Squirtle', 'Blossom', 'Ditto', 'Pikachu');
```

Comment:
[OPTIONAL] It would be better to use `WHERE animal_name = NULL`. In the scenario where there are hundreds or thousands of pokemons, you can't mention all of their names, it wouldn't be practical.